ccmbioinfo/lfs
# CARDS-2460: Compact layout for sections

**First attempt to a compact layout option for sections.**

Currently, if the section is set to have compact layout and the screen is wide enough, the entries should be displayed 3 per row (>1200px) or 2 per row (>900px). On narrower screens compact and non-compact look the same with one entry per row.

**To test:**
* Use the `Compact Sections Test` questionnaire
OR
* In Admin > Questionnaires, design a questionnaire containing a section with the `Compact` switch turned on
* Add questions to the section
* Preview the questionnaire to see the resulting layout

**Future work:**
* More flexible arrangement of entries in rows

**Known limitations:**
* The layout doesn't adjust well when there are hidden conditional entries in the compact section
* Compact sections inside compact sections probably look awful
* In patient portal, the form width is **at most 780px** - too narrow for 3/row, and even with 2/row it looks crowded for this type of user. For simplicity, the section's `compact` flag has no effect when the form is displayed in the patient portal.
